kubectl-pg/cmd/create.go

```diff
@@ -0,0 +1,61 @@
+// Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/zalando/postgres-operator/pkg/apis/acid.zalan.do/v1"
+	PostgresqlLister "github.com/zalando/postgres-operator/pkg/generated/clientset/versioned/typed/acid.zalan.do/v1"
+	"io/ioutil"
+	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/scheme"
+)
+
+// createCmd kubectl pg create.
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create command to create k8s postgres objects using manifest files",
+	Long: `Create command creates postgres custom resource objects from a manifest file.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		fileName,_ :=cmd.Flags().GetString("file")
+		create(fileName)
+	},
+}
+
+// Create postgresql resources.
+func create(fileName string) {
+	config := getConfig()
+	postgresConfig,err := PostgresqlLister.NewForConfig(config)
+	ymlFile,err := ioutil.ReadFile(fileName)
+	if err != nil {
+		panic(err)
+	}
+	decode := scheme.Codecs.UniversalDeserializer().Decode
+	obj,_,err := decode([]byte(ymlFile),nil, &v1.Postgresql{})
+	if err!=nil {
+		panic(err)
+	}
+	postgresSql := obj.(*v1.Postgresql)
+	_,err = postgresConfig.Postgresqls("default").Create(postgresSql)
```

Comment:
looks like using `.Postgresqls("default").Create` makes the plugin create a postgresql CR in the default namespace. If my manifest file has a different namespace defined I get this error:
```panic: the namespace of the provided object does not match the namespace sent on the request```

Reply:
seems to be the case everywhere except for the list command. The operator has a config parameter `watched_namespace`. I think, this could help here. So either a certain namespace or `""`

Reply:
Okay, I will read the namespace from manifest file and I will reference it during creation, updating, and deletion. But when I wanted to delete postgres cluster using the name. i.e
```kubectl pg delete -n acid-minimal-cluster``` this doesn't seem to work as we don't know which namespace this resource belongs to. I will look into this.

Reply:
but `acid-minimal-cluster` is the name of the postgres cluster, not the namespace. Or did you forgot the namespace in you command?

Reply:
We don't mention namespace anywhere in commands. @sdudoladov mentioned a command to delete the Postgres cluster using its name. So ```kubectl pg delete -n acid-minimal-cluster``` deletes the acid-minimal-cluster. I found a workaround for the same and just committed it. Let me know your views on it.